data/transactions/logic/opcodes.go

```diff
@@ -23,6 +23,11 @@ import (
 // LogicVersion defines default assembler and max eval versions
 const LogicVersion = 6
 
+// TODO: remove the following temporary constant when FIDO2 strategy finalized
+
+// LogicVersionInConsideration defines experimental assembler versions that are under consideration
+const LogicVersionInConsideration = 42
```

Comment:
I think I'd prefer this to be `LogicVersion+1`.

Perhaps the naming should be even more specific `const fidoVersion = LogicVersion + 1`

Then all three opcodes will be grouped, and it's easy to have a branch that turns them all on.  This might be a ncie way forward with other things, like exploring `splice` opcodes too.